helm/blueapi/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: blueapi
-description: A helm chart deploying a worker pod that runs Bluesky plans
+description: A Helm chart deploying a worker pod that runs Bluesky plans
```

Comment:
Could: If you're going to edit this then just remove the helm chart part altogether, as a user I know it's a helm chart, that is redundant.